ci_feature_prod/setup.sh

```diff
@@ -9,12 +9,12 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     dpkg-reconfigure --frontend=noninteractive locales && \
     update-locale LANG=en_US.UTF-8
 
-wget https://dockerprovider.blob.core.windows.net/omsagent/omsagent-1.8.1-416.universal.x64.sh
+wget https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.8.1.256/omsagent-1.8.1-256.universal.x64.sh
 
 #create file to disable omi service startup script
 touch /etc/.omi_disable_service_control
 
-wget "https://github.com/Microsoft/Docker-Provider/releases/download/4.0.0-0/docker-cimprov-4.0.0-0.universal.x86_64.sh"
+wget https://diliprstorage.blob.core.windows.net/mycontainer/docker-cimprov-3.0.0-5.universal.x86_64.sh
```

Comment:
can we change the version from 3.0.0.5 (which is last released version) to a different version please ?